card/spec/lib/card/diff_spec.rb

```diff
@@ -33,111 +33,111 @@ def p_diff
   describe 'traffic light' do
     it 'is green for addition' do
       a = "a"
-      b = "a b" 
+      b = "a b"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.